src/main/resources/application-qa.properties

```diff
@@ -0,0 +1,13 @@
+environment=QA
+###########################################################
+# Web application properties
+###########################################################
+host=https://www.onliner-qa.by/
+###########################################################
+# Files path properties
+###########################################################
+logPath=target/logs/appTest.log
+saveVideoPath=./Videos/
+###########################################################
+# Test data
```

Comment:
You can store valid log in and psw in property files

Reply:
OK